superset/reports/commands/execute.py

```diff
@@ -73,6 +74,8 @@
 from superset.utils.urls import get_url_path
 from superset.utils.webdriver import DashboardStandaloneMode
 
+from ...utils.core import HeaderDataType
```

Comment:
can you make this absolute instead of relative?

Reply:
yeah, I keep on doing this and the black reformats it. So annoying.

Reply:
+1 please fix this format

Reply:
yeah latest commit fixes it.